sklearn/gaussian_process/kernels.py

```diff
@@ -1704,18 +1744,15 @@ class PairwiseKernel(Kernel):
     """
 
     def __init__(self, gamma=1.0, gamma_bounds=(1e-5, 1e5), metric="linear",
-                 pairwise_kernels_kwargs=None):
+                 **pairwise_kernels_kwargs):
```

Comment:
Not sure either, but clearly gamma is to be a tunable hyperparameter, and all others are not.